src/boot/bless-boot-generator.c

```diff
@@ -0,0 +1,75 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <errno.h>
+#include <sys/stat.h>
+#include <unistd.h>
+
+#include "efivars.h"
+#include "log.h"
+#include "mkdir.h"
+#include "special.h"
+#include "string-util.h"
+#include "util.h"
+#include "virt.h"
+
+/* This generator pulls systemd-bless-boot.service into the initial transaction if the "LoaderBootCountPath" EFI
+ * variable is set, i.e. the system boots up with boot counting in effect, which means we should mark the boot as
+ * "good" if we manage to boot up this far. */
```

Comment:
"this far" is unclear. The generator runs very early, but the blessing happens very late.

Reply:
i changed this to "far enough" now. It's supposed to be vague, since after all it's supposed to open what is meant by "successful boot". But of course, the "this" is confusing indeed, since it's not just vague but potentially referencing something in the same paragraph and that's something we should fix which I hope "far enough" does. Hope that makes sense.